Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -192,3 +192,27 @@ public struct Aside {
         self.content = Array(newBlockQuote.blockChildren)
     }
 }
+
+extension BlockQuote {
+    /// Conservatively checks the text of this block quote to see whether it can be parsed as an aside.
+    ///
+    /// Whereas ``Aside/init(_:)`` will use all the text before the first colon in the first line,
+    /// or else return an ``Aside`` with a ``Aside/Kind-swift.struct`` of ``Aside/Kind/note``,
+    /// this function will allow parsers to only parse an aside if there is a single-word aside
+    /// marker in the first line, and otherwise fall back to a plain ``BlockQuote``.
+    func isAside() -> Bool {
```

Comment:
Should this be `public`?

Reply:
I feel like this API should be carefully considered. The existing Aside initializer is a bit more liberal with its parsing strategy in terms of the aside label, whereas i wanted to be more conservative in the HTML formatter. I think it might be more useful to introduce an optional initializer that allows for finer-grain selection of what counts as an aside label, so that you can optionally allow for multi-word labels, for example, or fall back to the existing permissive conversion where it will use a "Note" label if it doesn't parse one itself.